pallets/phala/src/compute/computation.rs

```diff
@@ -496,6 +498,101 @@ pub mod pallet {
 			ScheduledTokenomicUpdate::<T>::put(new_params);
 			Ok(())
 		}
+
+		#[pallet::weight(0)]
+		pub fn migrate_miners(
+			origin: OriginFor<T>,
+			max_iterations: u32,
+		) -> DispatchResult {
+			//ensure_root(origin)?;
```

Comment:
Looks good. Let's specify a special "operator" role (hard coded account) that can call the migration functions.

Reply:
Sure, will do it before final merge.